src/dodal/devices/util/motor_utils.py

```diff
@@ -0,0 +1,5 @@
+from ophyd import Component, EpicsMotor, EpicsSignalRO
+
+
+class EpicsMotorWithMRES(EpicsMotor):
```

Comment:
Could: make this `ExtendedEpicsMotor` or something like that, so that we can add `.ACCL` as well

Reply:
I think if we want to add that we can do so when we have a usecase. I'm also keen to move to `ophyd-async` before then, where we hopefully get these into the motor proper